public/app/core/components/AccessControl/PermissionList.tsx

```diff
@@ -3,23 +3,31 @@ import { ResourcePermission } from './types';
 import { PermissionListItem } from './PermissionListItem';
 
 interface Props {
-  title: string;
+  type: string;
```

Comment:
`type` is an overloaded term and I think it's quite confusing here, doesn't `title` work in this case? it seems like this prop is used as a title.

Reply:
Thanks I wasn't sure this change was for the best anyway, so i'll revert on that !